src/components/PhotoList.jsx

```diff
@@ -57,4 +57,24 @@ export class PhotoList extends Component {
   }
 }
 
-export default Dimensions()(PhotoList)
+export class PhotoLists extends Component {
+  render () {
+    const { photoLists, containerWidth, selected, onPhotoToggle } = this.props
+    return (
+      <div>
+        { photoLists.map(photoList => {
+          return (<PhotoList
```

Comment:
If `PhotoList` is not aimed to be used outside (instead of `PhotoLists`), it may not be exported.